homeassistant/components/overkiz/climate_entities/atlantic_electrical_heater_with_adjustable_temperature_setpoint.py

```diff
@@ -96,6 +99,16 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
     @property
     def preset_mode(self) -> str | None:
         """Return the current preset mode, e.g., home, away, temp."""
+
+        states = self.device.states
+
+        if (
+            (operating_mode := states[OverkizState.CORE_OPERATING_MODE])
+            and operating_mode.value_as_str
+            and operating_mode.value_as_str == OverkizCommandParam.EXTERNAL
```

Comment:
```suggestion
            (operating_mode := states[OverkizState.CORE_OPERATING_MODE])
            and operating_mode.value_as_str == OverkizCommandParam.EXTERNAL
```

Reply:
Why not apply this change?

Reply:
I did: https://github.com/home-assistant/core/pull/84010/files#diff-43828e0fbdf520a5d5d2352125ea58172464db83b046804b853f6f49a1424375R107